src/MetricExporter.ts

```diff
@@ -80,8 +77,8 @@ export class DynatraceMetricExporter implements MetricExporter {
 	}
 
 	export(
```

Comment:
Another nit: Most of the code is now tested in the utils package, but the `export` method still does quite some work (batching, actually creating the metric types, serializing then joining..) and no tests are covering those parts. I'm not sure how hard it would be to test it, since that would involve having to mock the httprequest, but I feel it's somewhat important?